conf.d/health.d/dockerd.conf

```diff
@@ -0,0 +1,8 @@
+template: docker_unhealthy_containers
+      on: docker.unhealthy_containers
+   units: unhealthy containers
+   every: 10s
```

Comment:
This is too strict. A lot of containerized services need much more time to bootstrap themselves before they become healthy. For example [minio container will need at least 30s](https://github.com/minio/minio/blob/a63bc9254d481a32f2abebd18fccbfa816c5f8bb/Dockerfile#L31)

Reply:
Docker `HEALTHCHECK` directive is in charge of setting the health of a container. https://docs.docker.com/engine/reference/builder/#healthcheck

As you can see, there is also a `starting` health status, so an `unhealthy` container went by `starting` status before. 

I don't think it's netdata "problem", it's an issue of the container configuration, no ?

Reply:
Of course it is not netdata problem. Problem is that you are setting very strict alert rule which will cause much problems on a lot of systems by sending false-positive alerts.

Reply:
But, which false-positive ? if docker says it's unhealthy, it's unhealthy, I don't see it as a false positive. 
What do you suggest ?

Reply:
I just ran some tests and seems that problem I was referencing is already fixed in docker daemon. Let's go with your alert setting implementation.